course02/homework00/yun/vector.h

```diff
@@ -0,0 +1,25 @@
+#include <array>
+#pragma once
```

Comment:
`#pragma once` should precede any includes or code - it doesnt make too much sense to include things before. This will likely not have any influences as in most cases those includes also use `#pragma once`, but it's a bit surprising / not recommended.